dspace-api/src/main/java/org/dspace/importer/external/service/ImportService.java

```diff
@@ -272,6 +296,35 @@ public ImportRecord getRecord(String uri, Query query) throws MetadataSourceExce
         return importSources.keySet();
     }
 
+    /*
+     * Get a collection of record from File,
+     * The first match will be return.
+     * 
+     * @param file  The file from which will read records
+     * @return a single record contains the metadatum
+     * @throws FileMultipleOccurencesException if more than one entry is found
+     */
+    public ImportRecord getRecord(File file) throws FileMultipleOccurencesException, FileSourceException {
+        ImportRecord importRecords = null;
+        for (MetadataSource metadataSource : importSources.values()) {
+            try (InputStream fileInputStream = new FileInputStream(file)) {
+                if (metadataSource instanceof FileSource) {
+                    FileSource fileSource = (FileSource)metadataSource;
+                    importRecords = fileSource.getRecord(fileInputStream);
+                    break;
+                }
+            } catch (FileSourceException e) {
```

Comment:
This exception is thrown in the getRecord() method if the provided file doesn't match the format for that source. If you have for example 5 sources & the last source is a match than 4 exceptions will be thrown and caught here. 
This would mean that the "exception" isn't really an exception but the most common output. 

I would remove this catch & either add a "supports()" or an "isValidFileForSource()" method (or something similar) to the FileSource class. This way you can validate a file before you attempt to parse it.